_webi/test.js

```diff
@@ -4,26 +4,26 @@
 // Print help if there's no pkgdir argument
 //
 var usage = [
-  'Usage: node _webi/test.js --debug <path-to-package>',
-  'Example: node _webi/test.js --debug ./node/'
+    'Usage: node _webi/test.js --debug <path-to-package>',
```

Comment:
This PR changes almost every line of the file. See my note in the comment section of this PR.